src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx

```diff
@@ -7,58 +7,66 @@ import { normalizeFullName } from '../../utils/helpers';
+  const { veteranFullName, veteranDateOfBirth, totalDisabilityRating } = user;
+  const veteranDOB = veteranDateOfBirth
+    ? formatDate(veteranDateOfBirth, 'MMMM dd, yyyy')
+    : null;
+  const veteranName = normalizeFullName(veteranFullName, true);
   return (
     <div className="vads-u-margin-top--2p5 vads-u-margin-bottom--2">
-      <p data-testid="hca-veteran-profile-intro">
-        {veteranDOB
-          ? 'This is the personal information we have on file for you.'
-          : 'Here\u2019s the name we have on file for you.'}
-      </p>
-      <div className="va-address-block vads-u-margin-y--3">
-        <dl>
-          <div data-testid="hca-veteran-fullname">
-            <dt className="vads-u-visibility--screen-reader">Full name:</dt>
-            <dd
-              className="vads-u-font-weight--bold dd-privacy-mask"
+      <p>This is the personal information we have on file for you.</p>
+      <va-card data-testid="hca-profile-card" background>
+        <ul className="hca-list-style-none">
+          <li>
+            <span className="vads-u-visibility--screen-reader">Full name:</span>{' '}
+            <span
+              className="vads-u-font-weight--bold vads-u-margin-bottom--1 dd-privacy-mask"
               data-dd-action-name="Veteran name"
             >
               {veteranName}
-            </dd>
-          </div>
+            </span>
+          </li>
           {veteranDOB ? (
-            <div data-testid="hca-veteran-dob">
-              <dt className="vads-u-display--inline-block vads-u-margin-right--0p5">
-                Date of birth:
-              </dt>
-              <dd
-                className="vads-u-display--inline-block dd-privacy-mask"
-                data-dd-action-name="Veteran date of birth"
+            <li>
+              <span>Date of birth:</span>{' '}
+              <span
+                className="dd-privacy-mask"
+                data-dd-action-name="Date of birth"
               >
                 {veteranDOB}
-              </dd>
-            </div>
+              </span>
+            </li>
           ) : null}
-        </dl>
-      </div>
+          <li>
+            And our records show that you have a{' '}
+            <strong>
+              VA service-connected disability rating of{' '}
+              <span
+                className="dd-privacy-mask"
+                data-dd-action-name="Disability rating"
+              >
+                {totalDisabilityRating}%
+              </span>
+              .
+            </strong>
+          </li>
+        </ul>
+      </va-card>
       <p>
         <strong>Note:</strong> If you need to update your personal information,
         call our VA benefits hotline at{' '}
         <va-telephone contact={CONTACTS.VA_BENEFITS} /> (
         <va-telephone contact={CONTACTS[711]} tty />
-        ), Monday through Friday, 8:00 a.m. to 9:00 p.m.{' '}
+        ). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m.{' '}
         <dfn>
```

Comment:
Could this be replaced with our `Abbr` component?

Reply:
Yes, definitely! I want to do that, as its own PR, as a global find and replace in the application as a whole.